docs/DeveloperGuide.md

```diff
@@ -731,3 +731,84 @@ testers are expected to do more *exploratory* testing.
    1. _{explain how to simulate a missing/corrupted file, and the expected behavior}_
 
 1. _{ more test cases …​ }_
+
+### Adding a guest
+
+1. Adding a guest
+
+   1. Prerequisites: Add a guest use add command, only one adding guest
+      the name of the adding guest should not in the guest book
+      Every part of guest information should be valid
```

Comment:
Can we use this phrasing instead:

"Prerequisites: Only one guest to be added; the name of the guest to be added should not be already in the guest book; the format and data of the command should be valid."

Reply:
fixed